spec/pundit_spec.rb

```diff
@@ -409,7 +409,7 @@
     end
 
     it "raises an error when the given record is nil" do
```

Comment:
Do we need this test at all if we allow the nil policy? Isn't it just testing now that it raises if the permission check fails, which is what we do in the test on line 397?

Reply:
I agree that this is now simply testing the NilPolicy, but it could also help prevent a re-implementation of #251 by documenting the intention. I suppose it could read
```ruby
it "denies authorization to destroy a nil object" do
```
or similar.

There's another argument to suggest we shouldn't test the nil case at all, based on the principle that `nil`, like any other object, should be allowed a policy (or lack of one) which functions however the implementer intends.

Reply:
Yeah, I meant we could remove it all together since now we treat `nil` as any other object. But let's keep it for now.